src/include/homeobject/pg_manager.hpp

```diff
@@ -32,6 +32,7 @@ struct PGInfo {
     pg_id_t id;
     mutable MemberSet members;
     peer_id_t replica_set_uuid;
+    homestore::uuid_t index_table_uuid;
```

Comment:
Should be part of PG; not PGInfo.

Reply:
or at least hidden somehow from StorageManager.

Reply:
Yes; but not memory or StorageManager. Let's look at this some more, it shouldn't need to be here and can be encapsulated below.

Reply:
Agree. Removed it from PGInfo.